lib/assertions.js

```diff
@@ -673,28 +673,42 @@ export default class DOMAssertions {
   }
 
   /**
+   * Finds a valid HTMLElement from target, or pushes a failing assertion if a valid
+   * element is not found.
    * @private
+   * @returns a valid HTMLElement, or null
    */
   findTargetElement() {
-    if (this.target === null) {
-      let message = `Element <unknown> exists`;
+    let el = this.findElement();
+
+    if (el === null) {
+      let message = `Element ${this.target || '<unknown>'} should exist`;
       this.pushResult({ message, result: false });
       return null;
     }
 
-    if (typeof this.target === 'string') {
+    return el;
+  }
+
+  /**
+   * Finds a valid HTMLElement from target
+   * @private
+   * @returns a valid HTMLElement, or null
+   * @throws TypeError will be thrown if target is an unrecognized type
+   */
+  findElement() {
```

Comment:
Since this is private, should it be underscored (I'm not sure what the norm is in qunit-dom)?

Also, is there a name that better describes the split between this method and `findTargetElement`?  Maybe `findTargetElementWithoutAssertion` (yeah, I know its pretty long)?

Reply:
Ya I went with no underscore as this repo had set the precedent of no underscore. FWIW, I prefer underscore...

Reply:
I can rename the new method, but it does seem a bit backwards. The `findElement` method is the simple case, and `findTargetElement` is the method that's layering on functionality. As such, I would likely want to rename `findTargetElement` to something like `findElementWithAssertion` rather than the opposite (renaming the smaller, more well defined method).